torch/onnx/verify.py

```diff
@@ -0,0 +1,262 @@
+import copy
+import io
+import os
+import tempfile
+from contextlib import ExitStack
+from typing import Optional, Union
+
+import numpy as np
+
+import torch
+from torch import Tensor
+from torch.onnx.utils import unpack_quantized_tensor
+
+_ORT_PROVIDERS = ["CPUExecutionProvider"]
+
+
+def _flatten_tuples(elem):
+    flattened = []
+    for t in elem:
```

Comment:
Avoid names that are too short: https://google.github.io/styleguide/pyguide.html#3161-names-to-avoid

Reply:
feels alright falling under
```
single character names, except for specifically allowed cases:

* counters or iterators (e.g. i, j, k, v, et al.)
```

Reply:
ok